bench/workgen/runner/workgen_perf_check.sh

```diff
@@ -0,0 +1,37 @@
+#! /bin/bash
+
+ERROR=0
+OUTPUT=output.log
+TEST=many-dhandle-stress.py
+echo python3 $TEST 2> $OUTPUT
+python3 $TEST 2> $OUTPUT
+
+# Check exceptions
+if grep -io "exception" $OUTPUT; then
+	echo ERROR
+	ERROR=1
+fi
+
+# Check if one DROP operation took too long
+THRESHOLD=45
```

Comment:
The latencies have largely exceeded this value in the recent build...
http://build.wiredtiger.com:8080/job/WT-7616-plots/plot/

Reply:
As discussed with @sulabhM, instead of hardcoding thresholds, we could get them from arguments.
For now, we need more runs of the Jenkins jobs to define them.